connector/src/main/java/org/geysermc/connector/utils/ItemUtils.java

```diff
@@ -58,4 +58,22 @@ public static int getCorrectBedrockDurability(GeyserSession session, int javaId,
         }
         return original;
     }
+
+    /**
+     * @param itemTag the NBT tag of the item
+     * @return the custom name of the item
+     */
+    public static String getCustomName(CompoundTag itemTag) {
+        if (itemTag != null) {
```

Comment:
Same as before, could this be split into `if (itemTag == null) return null;` to avoid an indention below?

Reply:
I think as-is is fine; we've been avoiding one-line if statements for a bit.

Reply:
Ah then my above suggestion might be a 😰